get-latest-tgf.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+get_local_tgf_version () {
+        TGF_LOCAL_VERSION=$(tgf --current-version | awk -F\  '{print $2}')
+}
+
+get_latest_tgf_version () {
+    if [ -z "$GITHUB_TOKEN" ]
+    then
+        echo 'GITHUB_TOKEN is not set.'
+    fi
+    TGF_LATEST_VERSION=$(curl --silent https://api.github.com/repos/coveo/tgf/releases/latest?access_token=${GITHUB_TOKEN} | grep tag_name | awk -F\" '{print $4}')
```

Comment:
Having an invalid GITHUB_TOKEN breaks your TGF installation completely. If `TGF_LATEST_VERSION` is empty or invalid, we should exit and stop there with an error message

Reply:
I was thinking about a solution to validate the token.
But this seems like a good one, thanks !

Should it `exit 1` ?

Reply:
That would be fine